backend/main/views_with_api.py

```diff
@@ -373,14 +373,19 @@ def get_step_table(request):
         json_data = []
         
         if run.current_step is not None:
-            for dataframe in dataframes:
-                if dataframe in run.current_outputs:
-                    data = run.current_outputs[dataframe]
+            for key in run.current_outputs:
+                if key[0] in dataframes:
+                    data = key[1]
```

Comment:
hier würde finde ich die bezeichnung key etwas verwirrend
```suggestion
            for key, value in run.current_outputs:
                if key in dataframes:
                    data = value
```